### PR TITLE
AO3-4489 Changes from legal

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4489

### CODE REVIEW

**Comment by scottsds** [comment]
Looks good.

File: app/views/home/tos_faq.html.erb [hunk 1]
```diff
@@ -413,6 +413,8 @@ Please use our search functions for this rather than creating a separate work.</
 			<h5 class="heading" id="collections_def">What do you mean by "collections"?</h5>
 			<p>Collections are groups of works collected together under one heading. Collections can be fic or art fests, exchanges, 'big bangs' matching artists, authors, and/or podficcers, or other types of creative challenges, as well as simple collections of fanworks chosen by the collection maintainer. <a href="https://archiveofourown.org/archive_faqs/18">Learn more about collections.</a>
 			</p>
+                        <h5 class="heading">What information can a challenge maintainer see about participants?</h5>
+                        <p>A challenge or gift exchange maintainer can see prompts as well as the username and email address that participants use to sign up, in case the maintainer needs to communicate with participants. Other collection maintainers cannot see email addresses by virtue of having a user’s work in their collections.</p>
```
Comment (sarken) [inline app/views/home/tos_faq.html.erb]: Really tiny annoying nitpick: "user’s work" has a smart/curly apostrophe when we use straight '

File: app/views/home/tos_faq.html.erb [hunk 2]
```diff
@@ -420,7 +422,7 @@ Please use our search functions for this rather than creating a separate work.</
 			<p>Unless there's an independent violation of the Content Policy, we won't intervene in collection decisions, even if they are arbitrary, biased, or wrong. You may want to add tags to your fanwork that will be of interest to people who are fans of relevant collections.
 			</p>
 			<h5 class="heading">How can I start a collection?</h5>
-			<p><a href="https://archiveofourown.org/archive_faqs/25">Please consult our Tutorials.</a></p>
+			<p><a href="https://archiveofourown.org/archive_faqs/25">Please consult our Tutorials.</a> Please note that participants may provide information to the maintainer for purposes of participating in a collection or challenge. Any use of this information other than to manage the collection or challenge is a violation of our Terms of Service and can result in the termination of the maintainer’s account.</p>
```
Comment (sarken) [inline app/views/home/tos_faq.html.erb]: Also tiny annoying nitpick: "maintainer’s account" has a smart/curly apostrophe when we use straight '

File: app/views/home/tos_faq.html.erb (hunk 1 above)
Comment (sarken) [inline app/views/home/tos_faq.html.erb]: The apostrophe in "maintainer's" got changed, but this one in "user’s work" still needs to be straightened out.

**Pull request merged**